roles/dhcp-server/defaults/main.yml

```diff
@@ -18,3 +18,8 @@ domain: video.debconf.org
 #staticips:
 #  hosts:
 #  - For the layout of the staticips.hosts variable, see the staticips role
+#
+
+default_install_dev: sda
+default_install_tasks:
```

Comment:
We'd never want to customise these, right?

Reply:
good Q... hmm... 
if needed, sdb can be put in all the  boxes [] list.
same with tasks

at lca 2017, most of ryans's boxes needed sdb.

I like all the boxes to have ubuntu-desktop.

but yeah, it does seem like a needless setting that makes things harder to follow, debug etc.

Want me to hard code it?  (seems so wrong)

Reply:
The tasks seems unnecessary in our contexts. I'm fine with default_install_dev

Reply:
_tasks removed.